CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # 2.2.1 Release notes
 
+### Bugfixes
```

Comment:
It looks like @finnschiermer fixed this in #2373, but please address all review feedback before merging.